lib/zmq-kernel.coffee

```diff
@@ -306,30 +329,27 @@ class ZMQKernel extends Kernel
 
 
     destroy: ->
```

Comment:
On 07/08/16 08:40, Lukas Geiger wrote:
> In lib/zmq-kernel.coffee <https://github.com/nteract/hydrogen/pull/396#discussion_r73797331>:
> 
>> @@ -306,30 +329,27 @@ class ZMQKernel extends Kernel
>>  
>>  
>>      destroy: ->
> 
> I think we should add the |restart| option from |@shutdown|.

I'd find this confusing. Because of its name, `destroy` makes me think that we expect it be used as a class destructor.

I wouldn't expect a class instance to be usable after invoking its destructor (and so, the restart option wouldn't make sense).

Do we have any use for `destroy(restart)`?